item-spec/examples/digitalglobe-sample.json

```diff
@@ -74,7 +74,7 @@
     "sat:sun_azimuth_angle": 168.7,
     "eo:cloud_cover": 50,
     "sat:off_nadir_angle": 18.4,
-    "eo:azimuth": 34.5,
+    "sat:azimuth": 34.5,
```

Comment:
azimuth_angle

Reply:
Hmm, that's fixed on my side and all pushed, try refreshing?

Reply:
oh wait, no it's not...sorry, more fixes coming